core/src/main/scala/org/apache/spark/deploy/PythonRunner.scala

```diff
@@ -44,7 +44,7 @@ object PythonRunner {
       .orElse(sparkConf.get(PYSPARK_PYTHON))
       .orElse(sys.env.get("PYSPARK_DRIVER_PYTHON"))
       .orElse(sys.env.get("PYSPARK_PYTHON"))
-      .getOrElse("python")
+      .getOrElse("python3")
```

Comment:
This is rather a followup of https://github.com/apache/spark/commit/4f79b9fffd592a2f24e5e759d726232134bda409

Reply:
Then, shall we proceed as another PR for `SPARK-32447 Use python3 by default in pyspark and find-spark-home scripts` because this PR seems to be under discussion.

Reply:
Sure, makes sense. I'll create a separate PR tomorrow.